src/components/Checkout/InContext/InContextCheckout.vue

```diff
@@ -55,7 +55,8 @@
 		/>
 
 		<div
-			:class="`in-context-login ${isCorporateCampaignPage ? 'tw-text-right' : ''}`"
+			class="in-context-login"
+			:class="{ 'tw-text-right' : !isCorporateCampaignPage }"
```

Comment:
This component needs to import the new `isCorporateCampaignPage` method from the `checkoutUtils` file. That same method could be imported to the other components that also have the `isCorporateCampaignPage` method, centralizing that logic.